app/services/participant_outcomes/void.rb

```diff
@@ -10,11 +10,15 @@ class Void
     def void_outcome
       return unless voidable_outcome?
 
-      ParticipantOutcome.create!(
-        declaration:,
-        completion_date: declaration_date,
-        state: "voided",
-      )
+      ApplicationRecord.transaction do
+        outcome = ParticipantOutcome.create!(
+          declaration:,
+          completion_date: declaration_date,
+          state: "voided",
+        )
+        outcome.application.update!(participant_outcome_state: "voided")
```

Comment:
not sure we handle voided in the `participant_outcome_state` and should probably nullify, but let's discuss general approach